docs/package.json

```diff
@@ -10,12 +10,13 @@
     "files": [],
     "scripts": {
         "images": "imagemin '_site/assets/images' --out-dir='_site/assets/images'",
-        "watch:sass": "sass --watch --poll src/assets/scss:src/assets/css",
+        "watch:sass": "sass --watch --poll src/assets/scss:_site/assets/css",
         "watch:eleventy": "eleventy --serve --port=2023",
-        "build:sass": "sass --style=compressed src/assets/scss:src/assets/css --no-source-map",
+        "build:postcss": "postcss _site/assets/css --replace",
+        "build:sass": "sass src/assets/scss:_site/assets/css --no-source-map",
```

Comment:
Why have we considered removing the `--style=compressed` in building sass? and adding cssnano instead?

Reply:
I explain that in the PR description